app/Services/Api/ApiEntityService.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Services\Api;
+
+use App\Models\MiscModel;
+use App\Models\Campaign;
+
+class ApiEntityService
+{
+    /**
+     */
+    public function saveEntity(array $entity, MiscModel $class, Campaign $campaign)
+    {
+        $tags = [];
+
+        // Prepare the data
+        unset($entity['module']);
+        if (!empty($entity['entry'])) {
+            $entity['entry'] = nl2br($entity['entry']);
```

Comment:
It's safe to assume people will send html, so we dont need nl2br(), as that will cause some surprises to users and be inconsistent with the other apis